### DISCUSSION TITLE
請問日後更新能否也提供中英混合的csv檔？

### DISCUSSION THREAD

**Discussion by whtang906** [GpointChen/FFXIVChnTextPatch-GP#150] [General]
如題。
有見目前提供的csv是中日混合的版本，想說日後是否也能提供中英混合的版本呢？

**Comment by GpointChen** [comment]
考慮到英版用戶，已盡量將CSV中的日文剔除，如仍有日文殘留，可以至issue或discussion回報。

想要中英混雜的CSV，可以在執行漢化替代英文後，使用[修改過的saintcoinach](https://github.com/GpointChen/SaintCoinach/tree/hexcode)再輸出。
